repo/packages/C/calico/2/config.json

```diff
@@ -0,0 +1,245 @@
+{
+  "properties": {
+    "Framework Settings": {
+      "description": "Configuration options for Calico's DC/OS Framework.",
+      "properties": {
+        "framework-name": {
+          "type": "string",
+          "description": "The name of the framework.",
+          "default": "calico-install-framework"
+        },
+        "master": {
+          "default": "zk://master.mesos:2181/mesos",
```

Comment:
We should probably switch this over to the new zk string.

`zk://zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4:2181,zk-5:2181/mesos`